### DISCUSSION TITLE
Define name for Stationlist as menu option

### DISCUSSION THREAD

**Discussion by andimik** [PE5PVB/TEF6686_ESP32#236] [Ideas]
Currently, the *Stationlist* server name is hard-coded to `TEF_tuner` on various places in the source code. In case you have more than one TEF, each has different IP addresses in the network, but still same name, which makes it difficult to identify. Of course you can change the source code for each device, if you want.

My idea is to add a menu option to define the name (just like a suffix `TEF_tuner1` till `TEF_tuner9` ) with the rotary button (like you need to do with the *Stationlist* client IP address).

**Comment by PE5PVB** [comment]
I've added an ID selector in the menu to give your tuner an ID between 1-10
